website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
Ohh, I get what you're saying -- I think we could do that. Although we would probably still want the client-side validation so that the user can't enter a lot of text and then have it rejected after submission. What do you think?